### ISSUE TITLE
Profile name must not be null when trying to use Label Fonts

### ISSUE THREAD

**Issue by okay1204** [stefvanschie/IF#588]
Full traceback:
```
Caused by: java.lang.NullPointerException: Profile name must not be null
        at java.util.Objects.requireNonNull(Objects.java:233) ~[?:?]
        at com.mojang.authlib.GameProfile.<init>(GameProfile.java:31) ~[authlib-5.0.47.jar:?]
        at me.screescree.inventoryframework.util.SkullUtil.setSkull(SkullUtil.java:54) ~[?:?]
        at me.screescree.inventoryframework.util.SkullUtil.getSkull(SkullUtil.java:41) ~[?:?]
        at me.screescree.inventoryframework.font.CSVFont.lambda$new$1(CSVFont.java:46) ~[?:?]
        at java.util.stream.Collectors.lambda$uniqKeysMapAccumulator$1(Collectors.java:180) ~[?:?]
        at java.util.stream.ReduceOps$3ReducingSink.accept(ReduceOps.java:169) ~[?:?]
        at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625) ~[?:?]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?]
        at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921) ~[?:?]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
        at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682) ~[?:?]
        at me.screescree.inventoryframework.font.CSVFont.<init>(CSVFont.java:46) ~[?:?]
        at me.screescree.inventoryframework.font.util.Font.<clinit>(Font.java:29) ~[?:?]
        at me.screescree.SuperiorSteed.superiorhorse.horseeditor.HorseEditor.<init>(HorseEditor.java:71) ~[?:?]
        at me.screescree.SuperiorSteed.superiorhorse.horseeditor.HorseEditor.<init>(HorseEditor.java:51) ~[?:?]
        at me.screescree.SuperiorSteed.commands.SummonHorse.onCommand(SummonHorse.java:49) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-api-1.20.2-R0.1-SNAPSHOT.jar:?]
```
`Label submitLabel = new Label(8, 4, 1, 1, Font.LIME);`

Hi, it looks like my Labels using Fonts are not working anymore, and I'm guessing this may be because of all the unmigrated Java accounts being purged by Microsoft. If this is the case, will this be fixed in the future?

**Comment by quiquelhappy** [comment]
the API should update, it must use the new playerProfile api: skullMeta.setPlayerProfile(this.plugin.getServer().createProfile(player.getUniqueId(), player.getName()));

**Comment by quiquelhappy** [comment]
@stefvanschie

**Comment by GregorSomething** [comment]
#814
That should fix it.

**Comment by stefvanschie** [comment]
This will be fixed in version 0.10.14.
